at-quannguyen/W4/bbcnews.rb

```diff
@@ -0,0 +1,29 @@
+require './channel.rb'
+class BbcNews
+  include Channel
+  extend Channel
+  attr_accessor :id, :title, :content, :category
+  @@list_bbcnews = Hash.new
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/f8dbe605eafc53803b1d102f4a00dad2a5e916b9/at-quannguyen/W4/bbcnews.rb#L6"></span>Style/EmptyLiteral: Use hash literal <code>{}</code> instead of <code>Hash.new</code>. (https://github.com/rubocop-hq/ruby-style-guide#literal-array-hash)</td>
    </tr>
  </tbody>
</table>